dev/PushNotifications/PushNotificationsLongRunningTask/NotificationListenerManager.cpp

```diff
@@ -37,15 +38,22 @@ void NotificationListenerManager::RemoveListener(std::wstring appId)
 {
     THROW_HR_IF(E_INVALIDARG, appId.empty());
 
-    auto lock = m_lock.lock_exclusive();
+    auto lock{ m_lock.lock_exclusive() };
 
     LOG_IF_FAILED(PushNotifications_UnregisterNotificationSinkForFullTrustApplication(appId.c_str()));
 
     m_notificationListeners.erase(appId);
 }
 
+bool NotificationListenerManager::HasSinkForAppId(std::wstring const& appId)
```

Comment:
Please change the name to something like `HasListenerForAppId`. Let's avoid mixing terminology when possible.

Reply:
We don't need this function anymore since we have GetAppIdentifier.